CONTRIBUTING.md

```diff
@@ -0,0 +1,466 @@
+requires just reference documentation or that you need reference, conceptual, and how to
+documentation.
+
+### Reference documentation
+
+Reference documentation should provide information about parts of a system including, but not
+limited to APIs and CLIs. The style of reference documentation should be the same for all reference
+documentation of that type. For example, API documentation should define all of the API's parameters,
+indicate if a parameter is required or optional, and show examples of the use of the API. These
+examples should be very generic and simple. If you feel like you need a more elaborate example,
+consider creating a procedural document to reinforce your reference documentation.
+
+### Conceptual documentation
+
+Conceptual documentation should try to be brief and for the most part should not go above 1 page.
+If you need to write more than one page to describe a concept, consider breaking that concept into
+sub-concepts by using headings. By keeping your document brief you achieve the following:
+
+- You do not overwhelm your reader with a wall of text.
+
+- You avoid losing the reader while they read your document.
+
+The first paragraph should try to be a brief summary of your document, this should allow the user to
+quickly read through it, determine what the document covers, and if this is relevant to what they
+want to learn. If your document has multiple headings, you should include a bulleted list with the
+high-level headings after this first paragraph.
+You should use graphics, images, or diagrams to reinforce certain concepts. The text that comes
+before and after the graphic should explain what the graphic shows. Images should be saved in
+a feature specific `images/` directory or a common `images/` directory. You should also save
+the source file of your images in an `images/src/` directory.
+
+Good conceptual documentation usually includes:
+
+- **Description** rather than instruction
+
+- **Background** concepts
+
+- **Diagrams** or other visual aids (preferably in .png format)
+
+- **Links** to how-to and/or reference docs
+After writing your document, it is good practice to proofread the document, put yourself in the
+user's shoes (no longer being the expert that developed the feature), and try to answer these
+questions:
+
+- Does the information in the document explain the concept completely?
+
+- Is there information that is not needed for this concept? If so, remove it.
+
+- Is there unnecessary detail about how things might work in the background?
+
+- If I am the user, is there additional information I would have liked to know?
+
+Then, add your feedback into your document.
+
+### Procedural documentation
+
+Procedural documents are divided into two categories:
+
+- **How-to** - Consider writing a how-to when you want to help the user accomplish a very specific
+  goal.
+
+- **Tutorial** - Consider writing a tutorial when you want to help the user learn about a bigger goal
```

Comment:
workshop?

Reply:
I would be a fan of changing the use of "workshop" to "tutorial".

To me, a tutorial is a guide for doing something, a workshop is doing a tutorial live with people.

"We write tutorials, and present them at workshops"

Note the reason why it was called the `substrate-collectables-workshop` was due to the fact it was originally written for a particular workshop. It just kind of exploded from there.

Reply:
I agree with Shawn, during a workshop you will follow a tutorial. We should refer to them as tutorials on the devhub.